src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -199,6 +203,10 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleExit();
             }
 
+            if (commandResult.isLoadImage()) {
```

Comment:
Based on my suggestions, since `CommandResult` contains `Image` class, it is possible to do this:
Image image = commandResult.getImagePath();
handleLoadImage(image)

handleLoadImage(Image image) {
     strategyPanel.changeImageBackground(image.getImagePath())
}